tests/purchase_test.ts

```diff
@@ -2,38 +2,52 @@ import LoginPage from "../pages/loginPage";
 import ProductPage from "../pages/productPage";
 import CartPage from "../pages/cartPage";
 import CheckoutPage from "../pages/checkoutPage";
-import User from "../helpers/interfaces";
+import User, { Product } from "../helpers/interfaces";
+import productPage from "../pages/productPage";
```

Comment:
you can list your pages in config in `include` and then use inside of `async ({ I, current, somePage })`

Reply:
done